### PR TITLE
ruby: Add RubySmellBear

### PR DESCRIPTION
**Pull request by sils1297**
Fixes https://github.com/coala-analyzer/coala-bears/issues/430

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -4,3 +4,4 @@ ruby '2.1.5'
 gem "rubocop"
 gem "sqlint"
 gem 'scss_lint', require: false# require flag is necessary https://github.com/brigade/scss-lint#installation
+gem "reek"
```
Comment (mr-karan) [inline Gemfile]: Don't know, but is alphabetical order required here?

Comment (sils1297) [inline Gemfile]: it isn't alphabetically ordered anyway

File: bears/ruby/RubySmellBear.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.Result import Result
+from coalib.results.SourceRange import SourceRange
+
+
+@linter(executable='reek', use_stdin=True)
+class RubySmellBear:
+    """
+    Detect code smells in Ruby source code.
```
Comment (Adrianzatreanu) [inline bears/ruby/RubySmellBear.py]: how does it detect it :/ why no regex?

Comment (sils1297) [inline bears/ruby/RubySmellBear.py]: see below, process_output is overridden

**Comment by Adrianzatreanu** [comment]
ack 3d63bd8

**Comment by Adrianzatreanu** [comment]
ack 510def1

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**